libkbfs/config_local.go

```diff
@@ -873,6 +873,7 @@ func (c *ConfigLocal) MaybeStartTrace(
 	}
 
 	tr := trace.New(family, title)
+	tr.SetMaxEvents(25)
```

Comment:
The default was 10. When it reaches the maximum it merges events in the middle so that it says "(50 omitted events)". I bumped it up to 100 initially when I was emitting a trace line for every journal block put, but bumped it back down after I had `doBlockPuts` emit just one log.

I haven't run into any omitted lines yet, even with the long-running requests, but if we do, it's easy enough to bump it higher.